Installation/test/Installation/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ endmacro()
 
 find_package(CGAL REQUIRED COMPONENTS Core)
 
+include(${CGAL_USE_FILE})
```

Comment:
Same here: have you found out why that is necessary?

Reply:
After having spent so much time on arrangement I gave up on this one too (for now)